addons/l10n_fi/data/account_tax_template.xml

```diff
@@ -0,0 +1,479 @@
+            <field name="amount">24.0</field>
+            <field name="type_tax_use">sale</field>
+            <field name="account_id" ref="chart2830"/>
+            <field name="refund_account_id" ref="chart2830"/>
+            <field name="amount_type">percent</field>
+            <field name="tag_ids" eval="[(6,0,[ref('301-sales24')])]"/>
+        </record>
+        <record id="vat0triangulation" model="account.tax.template">
+            <field name="chart_template_id" ref="fi_chart_template"/>
+            <field name="name">VAT 0% Triangulation</field>
+            <field name="description">VAT 0% Triangulation</field>
+            <field name="amount">0.0</field>
+            <field name="type_tax_use">sale</field>
+            <field name="account_id" ref="chart2830"/>
+            <field name="refund_account_id" ref="chart2830"/>
+            <field name="amount_type">percent</field>
+            <field name="tag_ids" eval="[(6,0,[ref('333-triangulation-eu')])]"/>
+        </record>
+        <record id="triangulation_purchase" model="account.tax.template">
+            <field name="chart_template_id" ref="fi_chart_template"/>
+            <field name="name">Triangulation Purchase</field>
+            <field name="description">Triangulation Purchase</field>
+            <field name="amount">0.0</field>
+            <field name="type_tax_use">purchase</field>
+            <field name="account_id" ref="chart1770"/>
+            <field name="refund_account_id" ref="chart1770"/>
+            <field name="amount_type">percent</field>
+            <field name="tag_ids" eval="[(6,0,[ref('330-triangulation-purchase')])]"/>
+        </record>
+        <record id="purchase24construct3" model="account.tax.template">
+            <field name="chart_template_id" ref="fi_chart_template"/>
+            <field name="name">Purchase 24% Construct3</field>
+            <field name="description">Purchase 24% Construct3</field>
+            <field name="amount">24.0</field>
+            <field name="type_tax_use">purchase</field>
+            <field name="account_id" ref="chart1770"/>
+            <field name="refund_account_id" ref="chart1770"/>
+            <field name="amount_type">percent</field>
+            <field name="tag_ids" eval="[(6,0,[ref('307-purchases')])]"/>
+        </record>
+        <record id="vat0export" model="account.tax.template">
+            <field name="chart_template_id" ref="fi_chart_template"/>
+            <field name="name">VAT 0% Export</field>
+            <field name="description">VAT 0% Export</field>
+            <field name="amount">0.0</field>
+            <field name="type_tax_use">sale</field>
+            <field name="account_id" ref="chart2830"/>
+            <field name="refund_account_id" ref="chart2830"/>
+            <field name="amount_type">percent</field>
+            <field name="tag_ids" eval="[(6,0,[ref('304-sales0')])]"/>
+        </record>
+        <record id="vat0export" model="account.tax.template">
```

Comment:
@thecodebasesite same tax record...!!!!
https://github.com/odoo/odoo/pull/30943/files#diff-1db11eddb8611ce58313ff80229bde14R433
AND
https://github.com/odoo/odoo/pull/30943/files#diff-1db11eddb8611ce58313ff80229bde14R444